packages/website/src/components/RulesTable/index.tsx

```diff
@@ -68,14 +80,32 @@ function RuleRow({
             : undefined
         }
       >
-        {fixable ? '🔧\n' : '\n'}
-        {hasSuggestions ? '💡' : ''}
+        {fixable ? `${FIXABLE_EMOJI}\n` : '\n'}
+        {hasSuggestions ? SUGGESTIONS_EMOJI : ''}
```

Comment:
```suggestion
        {fixable ? FIXABLE_EMOJI : ''}<br />
        {hasSuggestions ? SUGGESTIONS_EMOJI : ''}
```

I believe it should do the same thing